sample-tests/build.gradle.kts

```diff
@@ -7,18 +10,32 @@ repositories {
 }
 
 dependencies {
-    testImplementation("org.codehaus.groovy:groovy-all:2.5.8")
+    testImplementation("org.codehaus.groovy:groovy-all:${GroovySystem.getVersion()}")
     testImplementation(gradleTestKit())
-    testImplementation("org.gradle:sample-check:0.9.0")
+    testImplementation("org.gradle:sample-check:0.12.6")
 }
 
-val copySamples by tasks.registering(Sync::class) {
-    from("../samples")
-    into("$buildDir/samples")
-}
+val snippetsDir = file("../samples")
+val processedSnippetsDir = file("$buildDir/samples")
+
+val tokens: Map<String, Provider<String>> = mapOf(
+    "test-retry-plugin-version" to provider {
+        PluginsVersionData.latestVersion("org.gradle", "test-retry-gradle-plugin")
```

Comment:
❓ This should test the locally built version, shouldn't it?

Reply:
It seems that's currently not possible with Exemplar: https://github.com/gradle/exemplar/issues/38

Reply:
I guess the intention is to test the example with the latest released public available version.